Source/missiles.cpp

```diff
@@ -1854,16 +1842,16 @@ void UpdateVileMissPos(MissileStruct &missile, Point dst)
 {
 	for (int k = 1; k < 50; k++) {
 		for (int j = -k; j <= k; j++) {
-			int yy = j + dst.y;
+			missile.position.tile.y = j + dst.y;
 			for (int i = -k; i <= k; i++) {
-				int xx = i + dst.x;
-				if (PosOkPlayer(Players[MyPlayerId], { xx, yy })) {
-					missile.position.tile = { xx, yy };
+				missile.position.tile.x = i + dst.x;
+				if (PosOkPlayer(Players[MyPlayerId], missile.position.tile)) {
 					return;
 				}
 			}
 		}
 	}
+
 	missile.position.tile = dst;

```

Comment:
This assignment is unnecessary (as long as the loop doesn't modify `missile.position.tile` before the check succeeds anyway) since the same assignment is set before UpdateVileMissPos is called.